docs/experiment/ppl/cmd/search.rst

```diff
@@ -0,0 +1,59 @@
+=============
+search
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| Using ``search`` command to retrieve document from the index. ``search`` command could be only used as the first command in the PPL query.
+
```

Comment:
Maybe we should also put the rule (that search command is mandatory and used as the first command) in syntax.rst so that users can get to know it more straightforward

Reply:
sure, add another section to describe the command order.